Source/WebKit/WebProcess/WebPage/mac/WKAccessibilityWebPageObjectMac.mm

```diff
@@ -227,13 +222,9 @@ - (NSValue *)accessibilityAttributeSizeValue
         Locker lock { m_cacheLock };
         return [NSValue valueWithSize:(NSSize)m_size];
     }
-#endif
+#endif // ENABLE(ACCESSIBILITY_ISOLATED_TREE)
 
-    return ax::retrieveAutoreleasedValueFromMainThread<id>([PROTECTED_SELF] () -> RetainPtr<id> {
-        if (!protectedSelf->m_page)
-            return nil;
-        return [NSValue valueWithSize:(NSSize)protectedSelf->m_page->size()];
-    });
+    return m_page ? [NSValue valueWithSize:(NSSize)m_page->size()] : nil;
```

Comment:
Do we need to do this cast. Don't we return the right thing by default?

Reply:
Yep, you're right! Removed in latest commit.